src/ConnectedKubernetes/ConnectedKubernetes.Autorest/custom/helpers/HelmHelper.ps1

```diff
@@ -125,6 +125,61 @@ function IsAmd64 {
         return $isSupport
     }
 }
+function Get-AzureHelmPath {
+    [Microsoft.Azure.PowerShell.Cmdlets.ConnectedKubernetes.DoNotExportAttribute()]
+    param(
+        [string[]]$ChildPaths
```

Comment:
I expect I am revealing my powershell ignorance...

we're passing in a regular string, right?  as in `Get-AzureHelmPath -ChildPaths $ValuesFile`

so how come this is now plural `ChildPaths` and an array?

Reply:
It can be a single string (which gets treated as a single value array) or a comma separated list of strings.

Reply:
but we don't ever want it to be anything other than a single string, do we?  Why is it desirable to treat it as maybe-an-array?

Reply:
You're right and I'll change it rather than leave unused code.  I was matching the (newer Powershell only!) Join-Path but since I moved to Azureness into the function , it no longer takes multiple extra paths.